game/tests.py

```
@@ -154,6 +155,50 @@ def test_email_failure_renders_error_and_removes_pending_user(self):
+        }
+    },
+)
+class ResendOtpRateLimitTest(TestCase):
+    """OTP resend cooldown should be enforced server-side per email."""
+
+    def setUp(self):
+        cache.clear()
+        self.user = User.objects.create_user(
+            username='pendingplayer',
+            email='pending@example.com',
+            password='StrongPass123!',
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)